app/components/GlobalError/GlobalError.js

```diff
@@ -13,14 +14,15 @@ class GlobalError extends React.Component {
 
   render() {
     const { error, clearError } = this.props
+    const errorMsg = errorToUserFriendly(error)
```

Comment:
why do we need to declare this const here? we can probably just render `errorToUserFriendly(error)` right?